host: "10.0.1.201"
port: 15012
timeout: 1
buffer_size: 8192
external_force_max: 3000
exponential_scale:  5.1
exponential_threshold: .5
autopilot_speed: 9999
autopilot_force: 65535
frequency_desired_actuation: 10
frequency_minimum_actuation: 1
frequency_desired_human_control: 100
